### DISCUSSION TITLE
Spanish translations

### DISCUSSION THREAD

**Discussion by donlaiq** [lllyasviel/Fooocus#827] [Show and tell]
Hi there,

I could contribute with the Spanish translations, but I'll need to ask you a few questions about the terms.
Can anybody tell me what are the meanings of "SAI" and "MRE" appearing in several options from the "Style" tab?
Eventually, I will need some extra clarification, but let's start easy :).

**Comment by xhoxye** [comment]
Represents the author of the style, without translation
SAI - Stability AI
MRE - [Fooocus-MRE](https://github.com/MoonRide303/Fooocus-MRE)

**Comment by donlaiq** [comment]
Hey @xhoxye, thanks for the previous message!

Here I have another questions/comments:

28: What's positive ADM? The ADM formalism from Arnowitt, Deser and Misner?
133, 217: Using the same prompt and the same seed, I'm getting different images, so I'll need to differentiate both styles, but why there are two options about styles like for example "Art Deco" and "Artstyle Art Deco"?
154: "Futuristic Futuristic"? A typo? Maybe "Futuristic Futurism"?
166: Is there a game called "Fighting Game", or it's about general fighting games?
169: Is there a game called "Strategy Game", or it's about general strategy games?
208: Neon Noir or Neo Noir?
236: What's "dmt"?
266: Mandola or Mandala?
279: The software application?
283: The software application?

I've tried the command including the "--language sp" flag, and it's working in my end. However, there are some missing strings in the original json file (so, the English version is showed in these cases):
- The string "Use via API"
- The placeholder where you need to write your prompt, "Type prompt here."
- Within the "Advanced" tab, the third slider, starting with "Refiner Switch At"

**Comment by donlaiq** [comment]
"Moving model to GPU" and "Step i/n in the 1-th Sampling" are also missing.

**Comment by xhoxye** [comment]
1. For the translation of the styles name, please read here [https:github.com/lllyasviel/Fooocus/tree/main/sdxl_styles](https://github.com/lllyasviel/Fooocus/tree/main/sdxl_styles) the prompts here are helpful for you to understand the translation. 
2. Regarding the missing strings, you can add them directly to your translated text, and most of the time the translations can be displayed

**Comment by pablootero-cl** [comment]
y la traducción bro, did you make the translation bro?

**Reply by donlaiq** [reply]
La traducción la terminé en noviembre de 2023, pero los administradores del proyecto tienen que hacer el merge para integrarla, y eso ya no depende de mí.

**Reply by pablootero-cl** [reply]
Genial gracias, toca remplazar el archivo mientras, copie tu codigo. Gracias por el aporte.
